sdk/tables/azure-data-tables/azure/data/tables/_table_service_client.py

```diff
@@ -213,7 +213,7 @@ def create_table_if_not_exists(self, table_name, **kwargs):
                 :end-before: [END create_table_if_not_exists]
                 :language: python
                 :dedent: 8
-                :caption: Deleting a table from the TableServiceClient object
+                :caption: Creating a table if it doesn't exist, from the TableServiceClient object
```

Comment:
Do we need to make any corrections to the async caption?

Reply:
It's correct there: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/tables/azure-data-tables/azure/data/tables/aio/_table_service_client_async.py#L220